Fix Set-Location examples. (#6480)
MicrosoftDocs/PowerShell-Docs: reference/6/Microsoft.PowerShell.Management/Set-Location.md

This command sets the current location to the root of the `Env:` drive. It uses the **PassThru**
parameter to direct PowerShell to return a **PathInfo** object that represents the `Env:\` location.

### Example 3: Set location to the current location in the C: drive

```powershell
PS C:\Windows\> Set-Location HKLM:\
PS HKLM:\> Set-Location C:
PS C:\Windows\>
```

> The first command sets the current location to the root of the `HKLM:` drive in the Registry

Reviewer: ```suggestion
The first command sets the location to the root of the `HKLM:` drive in the Registry
```

provider.

> The second command sets the current location to the `C:` drive in the FileSystem provider.

Reviewer: ```suggestion
The second command sets the location to the current location of the `C:` drive in the FileSystem provider.
```

> When the drive name is specified in the form `DriveName:` (without backslash), then the cmdlet sets

Reviewer: ```suggestion
When the drive name is specified in the form `<DriveName>:` (without backslash), the cmdlet sets
```

> the current location to the current location in the PSDrive.

Reviewer: ```suggestion
the location to the current location in the PSDrive.
```

> To get current location in the PSDrive use the `Get-Location` cmdlet with the **PSdrive** parameter.

Reviewer: ```suggestion
To get the current location in the PSDrive use `Get-Location -PSDrive <DriveName>`.
```
